crates/anvil/src/eth/backend/mem/storage.rs

```diff
@@ -233,6 +234,8 @@ impl BlockchainStorage {
             gas_limit: env.block.gas_limit.to::<u128>(),
             beneficiary: env.block.coinbase,
             difficulty: env.block.difficulty,
+            blob_gas_used: Some(env.tx.blob_hashes.len() as u128 * DATA_GAS_PER_BLOB as u128),
```

Comment:
this is not quite right because this uses the txenv.

this function is not very clean because initializing this with the env is a bit weird.

what this should do here is `env.block.blob_excess_gas_and_price.as_ref().map(|_| 0)`